airflow/providers/vertica/hooks/vertica.py

```diff
@@ -17,9 +17,24 @@
 # under the License.
 from __future__ import annotations
 
+from typing import Any, Callable, Iterable, Mapping, overload
+
 from vertica_python import connect
 
-from airflow.providers.common.sql.hooks.sql import DbApiHook
+from airflow.providers.common.sql.hooks.sql import DbApiHook, fetch_all_handler
+
+
+def vertica_fetch_all_handler(cursor) -> list[tuple] | None:
+    """Replace the default DbApiHook fetch_all_handler ."""
+    to_return = fetch_all_handler(cursor)
```

Comment:
Could we rename `to_return`? Does `rows` make sense as the name of this value? or `result` might be a bit better than `to_return`